deployment-e2e/molecule/repo/converge.yml

```diff
@@ -3,6 +3,9 @@
   hosts: all
   become: true
   tasks:
+    - import_role:
+        name: ../../../deployment/roles/common
+        tasks_from: dependencies
```

Comment:
Need to add dependencies in this test scenario because otherwise there is no `git` installed on host machine